OpenRA.Renderer.Sdl2/ErrorHandler.cs

```diff
@@ -56,8 +56,10 @@ public static void WriteGraphicsLog(string message)
 			Log.Write("graphics",  "Vendor: {0}", GL.GetString(StringName.Vendor));
 			if (GL.GetString(StringName.Vendor).Contains("Microsoft"))
 			{
-				Log.Write("graphics", "Note:  The default driver provided by Microsoft does not include full OpenGL support.\n"
-					+ "Please install the latest drivers from your graphics card manufacturer's website.\n");
+				String msg = "";
```

Comment:
SA1118: The parameter spans multiple lines. If the parameter is short, place the entire parameter on a single line. Otherwise, save the contents of the parameter in a temporary variable and pass the temporary variable as a parameter.

Would you prefer the very long one-liner?